docs/src/pages/customization/palette/palette.md

```diff
@@ -155,27 +185,22 @@ Note that `contrastThreshold` follows a non-linear curve.
 
 Need inspiration? The Material Design team has built an awesome [palette configuration tool](/customization/color/#color-tool) to help you.
 
-## Type (light /dark theme)
+## Dark theme
 
 Material-UI comes with two theme variants, light (the default) and dark.
 
-You can make the theme dark by setting `type` to `dark`.
-While it's only a single property value change, internally it modifies the value of the following keys:
-
-- `palette.text`
-- `palette.divider`
-- `palette.background`
-- `palette.action`
```

Comment:
Why this particular change?

Reply:
It sounds too specific and not actionable. I'm happy to put it back if you think otherwise.

Reply:
The intention was to identify what light / dark theme colours are available to developers using the theme in their own components. If you think we should remove them, I'll tweak the wording of the preceding sentence.

Reply:
Interesting, I was wondering about adding new sections on the page to cover each of these keys, individually.

Reply:
I think that we need to better cover the capability of the theme.

Reply:
With #18776, it will no longer be true.

Reply:
Then perhaps the #18776 PR is a good place to resolve this, not here.

Reply:
I still haven't heard a good justification for removing these in this PR. ("We might document them differently in the future" doesn't count). Last chance before I put them back. 😉)

Reply:
Haha, feel free to put them back. I think that we could detail more important aspects of the palette. I'm not sure people are interested in this aspect, it could be noise.

Reply:
>  I'm not sure people are interested in this aspect, it could be noise.

Could be. Okay, lets leave the change (with the revise wording), and see what happens.